src/ethereum_test_tools/common/types.py

```diff
@@ -260,6 +260,29 @@ def must_be_equal(self, other: "Storage"):
                 raise Storage.KeyValueMismatch(key, 0, other.data[key])
 
 
+def storage_padding(storage: Dict) -> Dict:
+    """
+    Adds even padding to each storage element.
+    """
+    new_storage = {}
+    for addr, value in storage.items():
+        new_addr = addr.lstrip("0x").lstrip("0")
+        new_addr = "00" if new_addr == "" else new_addr
+        if len(new_addr) % 2 == 1:
+            new_addr = "0" + new_addr
+
+        new_value = value.lstrip("0x").lstrip("0")
+        new_value = "00" if new_value == "" else new_value
+        if len(new_value) % 2 == 1:
+            new_value = "0" + new_value
+
+        addr = "0x" + new_addr
+        value = "0x" + new_value
+        new_storage[addr] = value
+
+    return new_storage
```

Comment:
I think this section could be improved by introducing another extra function to perform the padding like `storage_key_value_padding`, which would do the stripping and the padding on a single key/value string.

Then we can use dictionary comprehension to make this a one-liner like so:
```
new_storage = { storage_key_value_padding(k):storage_key_value_padding(v) for (k,v) in storage}
```

Also it would be nice to have a small test for this specific function in `src/ethereum_test_tools/tests/test_types.py`.